### ISSUE TITLE
Add convenience function to enable antialiasing

### ISSUE THREAD

**Issue by kryoxide** [haskell-game/sdl2#109]
As far as i can see there is no easy way to enable antialiasing in the
documentation (as of yet). So maybe you could add something like the following:

``` haskell
-- | See https://wiki.libsdl.org/SDL_GLattr#multisample
enableAntialiasing :: IO ()
enableAntialiasing = do
  glSetAttribute SDL_GL_MULTISAMPLEBUFFERS 1
  glSetAttribute SDL_GL_MULTISAMPLESAMPLES 2
  return ()
```

I think, that a lot of users would like to see this feature and it takes a while to figure out if
you don't know these options beforehand.

**Comment by polarina** [comment]
This would ideally be integrated into `OpenGLConfig` in `SDL.Video.OpenGL`. These attributes have to be set before creating a window.

See https://github.com/haskell-game/sdl2/blob/dec93953fb925cdd291874a9c8c96ef2e078d788/src/SDL/Video.hs#L128.

You can also set multisampling by calling `SDL.Raw.Video.glSetAttribute` before creating your window.

**Comment by polarina** [comment]
This should be resolved with 7483007d2e83bc35f36a9ad4554d33b6f49f5f0d. Thanks!
